src/Service/ConfigurationService.php

```diff
@@ -136,10 +138,11 @@ public function getHmacKey()
      */
     public function getApiKey()
     {
-        if ($this->getEnvironment()) {
-            return $this->getApiKeyLive();
-        }
-
-        return $this->getApiKeyTest();
+        ```suggestion
```

Comment:
```suggestion
```

Reply:
👍  Fixed it locally, I'll push the new commits shortly.